connector/src/main/resources/bedrock/skin/geometry.humanoid.customskull.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Could this file be converted to use 4 spaces to fit with the rest of the project and the other geometry files?

Reply:
Fixed.